### PR TITLE
updating,show and delete user

### PR DESCRIPTION
**Pull request by ngyihkai**
Ready

### CODE REVIEW

**Comment by ngyihkai** [comment]
Ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -18,6 +24,27 @@ def create
...
+    @user = User.find_by(id: params[:id])
+    if @user.update_attributes(user_params)
+      flash[:success] = "Profile updated"
+      redirect_to @user
+    else
+      render "edit"
+    end
+  end
+
+  def destroy
```
Comment (huongnt-2545) [inline app/controllers/users_controller.rb:37]: With action destroy user, you need to check admin permission for user
Please refer the sample app for this

**Comment by ngyihkai** [comment]
Ready

**Review by huongnt-2545 — COMMENTED** [review]
Still the comment has not fixed yet, please check

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (ngyihkai) [reply app/controllers/users_controller.rb:38]: is it  need this  before_action :admin_user, only: :destroy to allow admin to log in first then destroy?

**Comment by ngyihkai** [comment]
Ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**